### DISCUSSION TITLE
Lesson 5: Unable to import brownie after it's installation

### DISCUSSION THREAD

**Discussion by Hrih** [smartcontractkit/full-blockchain-solidity-course-py#1845] [Q&A] [answered]
So here I installed brownie using 

> pip install eth-brownie

in a virtual environment in WSL2,
and  I got my "Hello!"  printed out successsfully.
However when it comes to importing brownie, I get 'Import "brownie" could not be resolved'.
I noticed that it also mentioned something about "isort", I definitely didn't install it. It probably has something to do with Python3.10.
I've taken a screenshot to show as many details as possible, 
and I'd be so grateful if anyone can help me with this!

![screenshot](https://user-images.githubusercontent.com/114748120/226103453-78b17eaa-d30e-465a-a5b4-68d9fb92e008.png)

**Comment by Hrih** [comment]
I restart VScode and reopen brownie-simple-storage folder, which I've tried several times a day ago and to no avail, it works this time, no squiggly line under the import statement! Still not sure what happened, though.

**Comment by cromewar** [comment, marked as answer]
Hello @Hrih 
The reason is because vscode not always gets the correct python interpreter from the console, so be sure you open it form the wsl command line using `code .`.

**Reply by Hrih** [reply]
Yeah... But if I got WSL in the bottom-left corner , that should mean I'm good to go, right?

**Reply by cromewar** [reply]
Not 100% you can manage your python instances inside WSL using something like `ven` or `virtualenv`, the easies way is to activate your environment from the wsl console and the open code with `code .`, also you can still select the python interpreter from the ones you have on WSL directly on VScode.
